SavingsManagementSystem.Service/User/Implementations/AdminService.cs

```diff
@@ -35,5 +44,41 @@ public async Task<RegistrationResponse> RegisterAsync(RegistrationRequest reques
 
 			return response;
 		}
+
+
+		public async Task SendMemberInviteAsync(string email)
+		{
+			//create a verification token for the link
+			var vToken = await _vTokenService.CreateVerificationTokenAsync(30, "Successful", email);
```

Comment:
you are only updating the status when mail sending is successful.. Note: it should be something like this "Mail sent successfully"

Reply:
in the try catch, i updated the database status as Fail if the mail failed to send.